reark/src/main/java/io/reark/reark/data/stores/StoreGetInterface.java

```diff
@@ -0,0 +1,10 @@
+package io.reark.reark.data.stores;
+
+import rx.Observable;
+
+/**
+ * Created by ttuo on 27/06/16.
+ */
+public interface StoreGetInterface<T, U> {
+    Observable<U> getStream(T id);
```

Comment:
get() might indeed be needed, but then it should return null if no value is found. I am not a huge fan of null values but then having a contains method that's async could not be used with getStream since the value could have disappeared between the calls. I guess get() is the only way to go.